packages/mantine/src/templates/DescriptionField.tsx

```diff
@@ -0,0 +1,23 @@
+import { DescriptionFieldProps, FormContextType, RJSFSchema, StrictRJSFSchema } from '@rjsf/utils';
+import { Text } from '@mantine/core';
+
+/** The `DescriptionField` is the template to use to render the description of a field
+ *
+ * @param props - The `DescriptionFieldProps` for this component
+ */
+export default function DescriptionField<
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>(props: DescriptionFieldProps<T, S, F>) {
+  const { id, description } = props;
+  if (description) {
+    return (
+      <Text id={id} mt={3} mb='sm'>
+        {description}
+      </Text>
+    );
+  }
+
```

Comment:
Use the new `RichDescription` from `core`
```suggestion
  const { id, description, registry, uiSchema } = props;
  if (description) {
    return (
      <Text id={id} mt={3} mb='sm'>
        <RichDescription description={description} registry={registry} uiSchema={uiSchema} />
      </Text>
    );
  }

```